chapters/introduction.tex

```diff
@@ -30,94 +39,116 @@ \section{Doing credible research at scale}
-and \textbf{big data} have important uses,
-primary data\sidenote{\textbf{Primary data:} data collected from first-hand sources.}
-is critical for answering specific research questions.\cite{levitt2009field}
-As the ambition of development researchers grows, so too has the complexity of the data
-on which they rely to make policy-relevant research conclusions from \textbf{field experiments}.\sidenote{\textbf{Field experiment:} experimental intervention in the real world, rather than in a laboratory.}
-Unfortunately, this seems to have happened (so far) without the creation of
-guidelines for practitioners to handle data efficiently and transparently,
-which could provide relevant and objective quality markers for research consumers.
-
-One important lesson we have learned from doing field work over this time is that
-the most overlooked parts of primary data work are reproducibility and collaboration.
-You may be working with people
-who have very different skillsets and mindsets than you,
-from and in a variety of cultures and contexts, and you will have to adopt workflows
-that everyone can agree upon, and that save time and hassle on every project.
-This is not easy. But for some reason, the people who agreed to write this book enjoy doing it.
-(In part this is because it has saved ourselves a lot of time and effort.)
-As we have worked with more and more DIME recruits
-we have realized that we barely have the time to give everyone the attention they deserve.
-This book itself is therefore intended to be a vehicle to document our experiences and share it with with future DIME team members.
-
-The \textbf{DIME Wiki} is one of our flagship resources designed for teams engaged in impact evaluation projects.
-It is available as a free online collection of our resources and best practices.\sidenote{\url{http://dimewiki.worldbank.org/}}
-This book therefore complements the detailed-but-unstructured DIME Wiki
-with a guided tour of the major tasks that make up primary data collection.\sidenote{Like this: \url{https://dimewiki.worldbank.org/wiki/Primary_Data_Collection}}
+at the World Bank's \textbf{Development Economics (DEC) Vice Presidency}.\sidenote{
+\url{https://www.worldbank.org/en/about/unit/unit-dec}}
+
+DIME generates high-quality and operationally relevant data and research 
+to transform development policy, help reduce extreme poverty, and secure shared prosperity. 
+It develops customized data and evidence ecosystems to produce actionable information 
+and recommend specific policy pathways to maximize impact.
+DIME conducts research in 60 countries with 200 agencies, leveraging a 
+US\$180 million research budget to shape the design and implementation of 
+US\$18 billion in development finance. 
+DIME also provides advisory services to 30 multilateral and bilateral development agencies. 
+Finally, DIME invests in public goods to improve the quality and reproducibility of development research around the world. 
```

Comment:
Mention that this book is one example of that?

Reply:
i thought that was implicit. but i could make it explicit if others think that's needed.

Reply:
done ed01f5f75fb8b5a0795b3c474e6f0452569c7c96